src/module/item/abstract-effect/document.ts

```diff
@@ -1,10 +1,40 @@
 import { ItemPF2e } from "@item";
+import { sluggify } from "@util";
 import { EffectBadge } from "./data";
 
 /** Base effect type for all PF2e effects including conditions and afflictions */
 export abstract class AbstractEffectPF2e extends ItemPF2e {
+    slugNormalized!: string;
```

Comment:
Maybe call this `rollOptionSlug` or the like and document as such

Reply:
I think the name is fine but its such a small thing so changed. Documentation was needed though.